frontend/src/views/BookingView.vue

```diff
@@ -41,7 +42,7 @@
     <booking-modal
       :open="showBookingModal"
       :event="selectedEvent"
-      :requires-confirmation="true"
+      :requires-confirmation="appointment?.booking_confirmation"
```

Comment:
If appointment is empty here I'm sure more things would break, but it also show that the booking was successfully booked. Should we just let it hard crash here to get some sentry information to fix any future bug?

Reply:
Good point. I removed the `?`.